src/ug_baro_sensor/sensor/setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import find_packages, setup
+
+package_name = 'sensor'
+
+setup(
+    name=package_name,
+    version='0.0.0',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='Anthony Cieri',
+    maintainer_email='penguinmillion@gmail.com',
+    description='TODO: Package description',
+    license='MIT',
+    tests_require=['pytest'],
+    entry_points={
+        'console_scripts': [
+            "baro_node=sensor.depth_sensor:main"
```

Comment:
For the sake of convention, I would like to change it to `"pressure_node = ug_sensors.pressure_sensor:main"`.